x/operator/client/cli/query.go

```diff
@@ -415,3 +417,39 @@ func QueryAllAVSsByOperator() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetOptInfo queries opt info
+func GetOptInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "GetOptInfo <operatorAddr> <avsAddr>",
+		Short:   "Get opt info",
+		Long:    "Get opt info of specified operator and AVS",
+		Example: "exocored query operator GetOptInfo  exo1c5x7mxphvgavjhu0au9jjqnfqcyspevtyy27mz 0xaa089ba103f765fcea44808bd3d4073523254c57",
+		Args:    cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return xerrors.Errorf("invalid operator address,err:%s", err.Error())
+			}
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryOptInfoRequest{
+				Details: &operatortypes.OperatorAVSAddressDetails{
+					OperatorAddr: args[0],
+					AVSAddress:   args[1],
```

Comment:
Please validate it to be a hex address and convert it to the expected lower/upper/checksum case

Reply:
Thanks, It has been fixed.